recipes/coinmp/meta.yaml

```diff
@@ -0,0 +1,46 @@
+{% set name = "CoinMP" %}
+{% set version = "1.8.3" %}
+{% set sha256 = "7ae0c37c15d09f91c7fda33729d930e612ded8120d96c3e0932d82b7127cc5d0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.zip
+  url: http://www.coin-or.org/download/source/CoinMP/{{ name }}-{{ version }}.zip
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  skip: True  # [win]
+
+requirements:
+  build:
+    - toolchain
+    - pkgconfig
+    - readline
+    - zlib
+    - bzip2
+  run:
+    - readline
+    - zlib
+    - bzip2
+
+test:
+  commands:
+    - cbc ?
+    - clp ?
```

Comment:
Do you have a nice example howto do so? Do you want a small program to be compiled?